blog/2023-10-10-weaviate-1-22-release/index.mdx

```diff
@@ -0,0 +1,253 @@
+    "properties": [
+        {
+            "dataType": ["object"],
+            "name": "parent",
+            "nestedProperties": [
+                {
+                    "dataType": ["object"],
+                    "name": "child",
+                    "nestedProperties": [
+                        {
+                            "dataType": ["text"],
+                            "name": "childname",
+                        },
+                        {
+                            "dataType": ["text"],
+                            "name": "grandchild"
+                        },
+                    ],
+                },
+            ],
+        }
+    ],
+}
+```
+
+An example object of this class may look like this:
+
+```json
+{
+    "parent": {
+        "child": {
+            "childname": "Nested child",
+            "granchild": "I'm doubly nested!"
+        }
+    }
+},
+```
+
+</details>
+
+More practically, this type of structure may be applied to capture addresses, product information, or any other nested data structure. Take a look at the following example, which shows a `Person` class with an `address` property that is an `object` datatype. The `address` property is able to store its own `nestedProperties` of `street` and `city`.
+
+```json
+{
+    "class": "Person",
+    "properties": [
+        {
+            "dataType": ["text"],
+            "name": "last_name",
+        },
+        {
+            "dataType": ["object"],
+            "name": "address",
+            "nestedProperties": [
+                {"dataType": ["text"], "name": "street"},
+                {"dataType": ["text"], "name": "city"}
+            ],
+        }
+    ],
+}
+```
+
+The user may then create an object of this class, such as:
+
+```json
+{
+    "last_name": "Franklin",
+    "address": {
+        "city": "London",
+        "street": "King Street"
+    }
+}
+```
+
+### Auto-schema support for nested properties
+
+Given that the `object` data types allow nested properties of arbitrary depth, you will be pleased to know that auto-schema supports nested properties. This means that you can create a class with nested properties without having to manually define the class schema.
+
+In other words - you can create an `object` property, and Weaviate will automatically create the nested properties for you at import time.
+
+As with before, you will get the most granular and exact control over the schema by defining it manually. However, auto-schema may save you a lot of time, for example if you have a large number of nested properties.
+
+### Future improvements
+
+Please keep in mind that this is the first iteration of nested JSON object storage in Weaviate, and we have further improvements planned.
+
+In the initial implementation, any properties with `object` datatype do not affect the object vector, or the inverted index. This behavior is similar to what would happen with a cross-reference property, where the cross-referenced object is not indexed.
```

Comment:
Consider 

initial implementation, properties

and

Cross-reference properties have similar behavior whe the cross-referenced object is not indexed.